settings.py

```diff
@@ -116,7 +116,13 @@ def int_or_float(string):
 
 
 def _settings():
-    return sublime.load_settings("PackageDev.sublime-settings")
+    """Return global package settings object or load it first if required."""
```

Comment:
> ST caches settings in its internal API already

I know. That's why I found creating a new python object on each call somehow a waste. This is what I meant with creating "garbage" as the objects need to be collected by the garbage collector again and again. I recently found this way to cache the object in one of @r-steins implementations and found it a quite interesting construct.